Lecture_1/task_1/src/main/java/main.java

```diff
@@ -0,0 +1,52 @@
+import java.util.Scanner;
+
+class task_1 {
+
+    public static void main(String[] args) {
+        String input = inputString();
+        reverseString(input);
+        everyWordOnNewLine(input);
+        replaced(input);
+        certainElements(input);
+        upperCase(input);
+    }
+
+    public static String inputString() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Input string: ");
```

Comment:
avoid duplication  - move common code to private method like `log(String message)`